packages/service-clients/odsp-client/src/odspAudience.ts

```diff
@@ -28,6 +28,12 @@ interface OdspUser {
 	oid: string;
 }
 
+/**
+ * Creates an audience member from an IClient instance.
+ *
+ * @param audienceMember - the client instance/
+ * @returns An {@link OdspMember}.
```

Comment:
Nit: I don't think this `@returns` block is necessary, since the type-system captures the same information.

Reply:
Removed in latest.